### DISCUSSION TITLE
App keeps freezing

### DISCUSSION THREAD

**Discussion by Razzorback225** [community/community#113545] [Mobile]
### Select Topic Area

Bug

### Body

Hi, 

I'm experiencing some issues with the mobile app. 
When I'm using it, it may work properly for like 2 minutes and then I cannot scroll down or tap on any of the button or link in the app as it is completely frozen.

I tried to reinstall the app but it still freeze and I'm not able to log in again.

I've just reinstalled the app and it seems to work properly now but I don't know how long it will stay like this.

**Comment by elisealix22** [comment]
Thanks for reaching out, @Razzorback225 ! Are you using Android or iOS? Also, do you know what screen you are on when it freezes? It sounds like the Home screen or one of the main tabs?

**Reply by Razzorback225** [reply]
Hello @elisealix22,

I forgot to mention the OS, my apologizes.

I'm on Android 11 (Build number RPDS31.Q4U-39-26-14-10-1)

Sometimes it's directly on the home screen, some times it is in my repositories or even in public repositories

**Reply by elisealix22** [reply]
Thank you so much for the extra information! We will look into this!
